whylog/front/utils.py

```diff
@@ -0,0 +1,19 @@
+LOCALHOST = '127.0.0.1'
+
+
+class FrontInput(object):
+    def __init__(self, offset, line_content, resource_location):
+        self.offset = offset
+        self.line_content = line_content
+        self.resource_location = resource_location
+
+
+class AbstractResourceLocation(object):
+    def __init__(self, path, host):
+        self.path = path
+        self.host = host
+
+
+class LocallyAccessibleLogOpener(AbstractResourceLocation):
+    def __init__(self, path):
+        (super(LocallyAccessibleLogOpener, self).__init__(path, LOCALHOST))
```

Comment:
Outer parenthesis are unnecessary. Hardcoding IP address here doesn't seem to be useful - access will be completely different than in case of remote log, that is why there is a separate class for those two in the first place